packages/vscode-extension/src/webviews/connectionPanel.ts

```diff
@@ -265,7 +276,7 @@ export class ConnectionPanel {
                     }" data-invalid="${this.authIsInvalid()}" />
                   </div>
                   <div class="form--actions">
-                    <input id="save-connection" type="submit" value="Save Connection" />
+                    <input id="save-connection" type="submit" value="Save & Connect" />
```

Comment:
More feedback from Greg: the button should say `Save & Connect` to make clear the action we are about to perform.

This is similar to the MongoDB extension.